portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicRecording.java

```diff
@@ -67,6 +72,10 @@ public List<MusicGroup> getByArtists() {
     return byArtists;
   }
 
+  public boolean getIsExplicit() {
```

Comment:
nit: wdyt of moving the javadoc here onto this public member?

Reply:
That makes more sense. Thank you for the advice!